examples/step-44/step-44.cc

```diff
@@ -714,11 +713,10 @@ namespace Step44
       Jc(SymmetricTensor<4, dim>())
     {}
 
+    // That the material is stored in a smart pointer, we don't need to worry
+    // about its memory management
```

Comment:
That -> since? I don't think I understand the grammar of the sentence otherwise. But in reality, I would just remove the comment. If there is nothing to comment about, don't comment :-)